src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -307,6 +311,7 @@ export class ZaakViewComponent extends AbstractView implements OnInit, AfterView
         if (event) {
             console.log('callback loadTaken: ' + event.key);
         }
+        this.websocketService.suspendListener(this.zaakTakenListener);
```

Comment:
Deze begrijp ik niet. Komt er een ongewenst websocket bericht binnen als je de taken laadt? Welke dan?

Reply:
Deze heb ik laten staan.
Als je zelf een taak toevoegt kreeg je voorheen een melding dat er een taak is toegevoegd. Dit is juist niet gewenst. Je wil alleen een melding krijgen als iemand anders een taak heeft toegevoegd.

Reply:
Dat begrijp ik niet. Dan moet je toch suspenden in de method waar die taak toegevoegd wordt. Niet in de method waar ze geladen worden, want dan krijgt niemand ze meer.